jina/types/document/generators.py

```diff
@@ -116,6 +118,89 @@ def from_csv(
             yield Document(value, field_resolver)
 
 
+def from_huggingface_datasets(
+    dataset_path: str,
+    field_resolver: Optional[Dict[str, str]] = None,
+    size: Optional[int] = None,
+    sampling_rate: Optional[float] = None,
+    filter_fields: Optional[bool] = False,
```

Comment:
a default boolean with a default other than None seems weird. Just have a boolean defaulting to `False`